### PR TITLE
2to3

### LINKED ISSUE #30: Support python 3
```python
In [1]: import medleydb
  File "/home/bmcfee/data/medleydb/medleydb/__init__.py", line 20
    the top level Audio folder for MedeleyDB."""
                                                
^
SyntaxError: Missing parentheses in call to 'print'
```

.. or at least do a ``from __future__ import print_function``.  But I suspect there are other less obvious gotchas kicking around.

(also `Medley` is misspelled :))

### PR DESCRIPTION
**Pull request by bmcfee**
Trying to resolve #30

### CODE REVIEW

**Comment by bmcfee** [comment]
The tests pass on this now, but the coverage is quite low, so it's hard to say that it's "correct".  However, I don't think this PR is the place to focus on ramping up test coverage either.

**Comment by bmcfee** [comment]
Additional general comments:

- There were a bunch of places with explicit casts to unicode.  2to3 turned these all into casts to string.  I suspect what we actually want is `six.u(my_string)`.
- There were tons of checks of the form `foo in my_dict.keys()`, which 2to3 turned into `list(my_dict.keys())`.  This is inefficient, since it builds a list of the keys and linear-searches for a match.  It's generally better to use `foo in my_dict`.  I made the replacements by hand, but may have missed some.

**Comment by bmcfee** [comment]
@rabitt I think this is as ready as it'll get for now.  Please review.

File: medleydb/annotate/generate_melody_annotations.py [hunk 1]
```diff
@@ -14,7 +18,7 @@
 def get_time_stamps(total_duration):
     time_stamps = []
     n_stamps = int(np.ceil((total_duration*FS)/HOP))
-    time_stamps = np.array(range(n_stamps))*(HOP/FS)
+    time_stamps = np.arange(n_stamps) * (HOP / FS)
```
Comment (rabitt) [inline medleydb/annotate/generate_melody_annotations.py]: :)

Comment (rabitt) [inline medleydb/annotate/generate_melody_annotations.py]: !completed

**Comment by rabitt** [comment]
lgtm. thanks for finding and fixing some unrelated derps as well :)

**Pull request merged**